src/blockchain/log-processors/order-created.ts

```diff
@@ -49,6 +49,8 @@ export async function processOrderCreatedLog(augur: Augur, log: FormattedEventLo
       fullPrecisionPrice: fullPrecisionPrice.toString(),
       fullPrecisionAmount: fullPrecisionAmount.toString(),
       originalFullPrecisionAmount: fullPrecisionAmount.toString(),
+      originalTokensEscrowed: fixedPointToDecimal(moneyEscrowed, BN_WEI_PER_ETHER).toString(),
```

Comment:
Small thing-- can we take the value `fixedPointToDecimal(moneyEscrowed, BN_WEI_PER_ETHER).toString()`, assign it to a variable, and use it to set both `originalTokensEscrowed` and `tokensEscrowed`?

Reply:
Sure thing